### PR TITLE
docs: Update .tool-versions with Node.js 20.11.1

### PR DESCRIPTION
**Pull request by RocketRene**
## What this PR does / why we need it:
This PR adds nodeJS to the .tools-versions for the asdf package manager that we use to build the docs

### CODE REVIEW

**Review by soerenmartius — APPROVED** [review]

File: docs/.tool-versions [hunk 1]
```diff
@@ -1 +1,2 @@
 pnpm 8.11.0
+nodejs 18.19.0
```
Comment (mariux) [inline docs/.tool-versions:2]: Please use newer version here.

**Review by mariux — DISMISSED** [review]

File: docs/.tool-versions (hunk 1 above)
Comment (mariux) [reply docs/.tool-versions:2]: the latest LTS please

Comment (soerenmartius) [reply docs/.tool-versions:2]: this would be `20.11.1`

**Review by soerenmartius — APPROVED** [review]
lgtm

**Review by mariux — APPROVED** [review]

**Pull request merged**